### DISCUSSION TITLE
Expected History and Review/Motion Scrubbing

### DISCUSSION THREAD

**Discussion by SamCWB2** [blakeblackshear/frigate#14183] [Ask A Question] [answered]
Hello!  New user here--thank you Blake and team for such a great project!

We just have a simple v14 Ubuntu installation with one camera.  Everything seems to be working and object detection is fantastic.

Maybe this is expected since we configured Frigate to only retain active objects (for 3 days) but should we be able to go back more than one hour when scrubbing backwards in camera history?   

We notice the same limit in the Review/Motion tab--we can only go back around one hour before we get "No Preview Found".

I'm guessing we need to enable another type of recording.  We didn't want to record all video because our machine has an old SSD, but being able to see the once a minute snapshots going back in time for the 3 days would be a great compromise.

We signed up for Plus and we are able to see snapshots (that are older than one hour) eligible for upload, so those are being saved.

Thanks for any advice!  

Sam

**Comment by NickM-27** [comment, marked as answer]
Yes, previews are only kept for hours that have recordings saved.

**Reply by SamCWB2** [reply]
OK I understand....thanks Nick
